ChildProject/pipelines/samplers.py

```diff
@@ -294,25 +296,23 @@ def _segment_scores(self, recording):
         windows = pd.merge(segment_onsets, segment_offsets, left_index=True, right_index=True).reset_index()
         windows['recording_filename'] = recording['recording_filename']
 
-        # NOTE: The original code explicitly chooses for these conversational turn 
-        # types, but we might wish to verify what they are. 
-        segments['is_CT'] = segments['lena_conv_turn_type'].isin(['TIFR', 'TIMR'])
+        if self.metric == 'ctc':
+            # NOTE: The original code explicitly chooses for these conversational turn 
+            # types, but we might wish to verify what they are. 
+            segments['is_CT'] = segments['lena_conv_turn_type'].isin(['TIFR', 'TIMR'])
 
-        # NOTE: This is the equivalent of CTC (tab1) in rlena_extract.R
-        ctc = segments.groupby('chunk', as_index=False)[['is_CT']].sum().rename(columns={'is_CT': 'ctc'}).sort_values(by='ctc', ascending='False').merge(windows)
-
-        # NOTE: This is the equivalent of CVC (tab2) in rlena_extract.R
-        cvc = segments[segments.speaker_type.isin(['OCH', 'CHI'])].groupby('chunk', as_index=False)[['utterances_count']].sum().rename(columns={'utterances_count': 'cvc'}).merge(windows)
-
-        # NOTE: This is the equivalent of AWC (tab3) in rlena_extract.R
-        awc = segments[segments.speaker_type.isin(['FEM', 'MAL'])].groupby('chunk', as_index=False)[['words']].sum().rename(columns={'words': 'awc'}).merge(windows)
+            # NOTE: This is the equivalent of CTC (tab1) in rlena_extract.R
+            return segments.groupby('chunk', as_index=False)[['is_CT']].sum().rename(columns={'is_CT': 'ctc'}).sort_values(by='ctc', ascending='False').merge(windows)
```

Comment:
The sort here is not necessary probably since we sort in the call to `sample` below.

Reply:
oops, indeed!